src/index.html

```diff
@@ -1,14 +1,38 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Moyo header</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Moyo header</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <header class="nav_logo">
+      <a href="link" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav_block">
+        <div class="each">
+          <a href="link" class="block is-active">APPLE</a>
+        </div>
+        <div class="each"><a href="link" class="block">SAMSUNG</a></div>
```

Comment:
```suggestion
        <div class="each">
          <a href="link" class="block">
            Samsung
          </a>
        </div>
```

Остальные блоки отформатируй так же